demisto_sdk/commands/lint/tests/test_linter/os_runner_test.py

```diff
@@ -80,41 +80,86 @@ def test_run_bandit_usage_stderr(self, linter_obj: Linter, lint_files: List[Path
 
 
 class TestMypy:
```

Comment:
since all tests under this class are for version 2.7
```suggestion
class TestMypy_2_7:
```